frame/mosaic/README.md

```diff
@@ -1,10 +1,60 @@
 # Mosaic
 
-Pallet implementing an interface for the Mosaic Relayer. As opposed to the EVM-EVM bridge, this pallet takes a different approach and uses `mint` and `burn` operations.
-Because of that it also limits the mintable amount by the relayer using a decaying penalty.
+The Mosaic pallet enables cross-chain and cross-layer transfers
 
-## Decaying Penalty
+## Overview
+
+The Mosaic Pallet implements the interface for the Mosaic Relayer. The Mosaic 
+Relayer will relay liqudity accross chains and layers.
+
+As opposed to the EVM-EVM bridge, this pallet takes a different approach and 
+uses mint and burn operations. Because of that it also limits the amount the 
+relayer can mint using a decaying penalty.
+
+### Decaying Penalty
+
+At moment N, the Relayer has a maximum budget budget. Minting a token adds a 
+penalty penalty to the Relayer. The penalty decreases each block according to 
```

Comment:
same here, maybe "penalty `penalty`" if you're referring to a variable of sorts?

Reply:
Nope, just me being blind
32b481f76332c3f5a1fa991893aa48fa4193f22a